### PR TITLE
Fix proximity command

### PR DESCRIPTION
**Pull request by Pablete1234**

### CODE REVIEW

File: src/main/resources/lang/en.xml [hunk 1]
```diff
@@ -100,6 +100,8 @@
         <invalidArguments>Invalid arguments.</invalidArguments>
         <tooFewArguments>Too few arguments.</tooFewArguments>
         <cannotCalculateScores>Team scores cannot be calculated.</cannotCalculateScores>
+        <proximityObsOnly>The /proximity command is only available to observers</proximityObsOnly>
```
Comment (SungMatt) [inline src/main/resources/lang/en.xml]: Proper English uses full stops.

**Comment by iEli2tyree011** [comment]
Why did you remove the proximity command permission?

**Comment by twizmwazin** [comment]
Can you re-add the permission?

**Comment by Pablete1234** [comment]
@twizmwazin done
@iEli2tyree011 i didn't remove it, it's just a copy-paste from the other PR (i splitted them), and it's how it was there

**Comment by Pablete1234** [comment]
@twizmwazin anything else that i should add here?

**Comment by twizmwazin** [comment]
merged

**Pull request closed without merging**